pkgs/tools/networking/jnettop/0001-Use-64-bit-integers-for-byte-totals-support-bigger-u.patch

```diff
@@ -0,0 +1,104 @@
+From: Ari Pollak <aripollak@gmail.com>
```

Comment:
Unfortunately `sources.debian.net` is not stable because it only covers currently released versions in Debian releases. I'm generally using a tag in Debian's git web interface if the packaging is done in a git repo. For this packages, that isn't the case and we have no other nice source for that patch, so we'll go with `sources.debian.net` here. Thanks! :+1: